Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -1,19 +1,20 @@
 #include "BatteryStateOfChargeService.h"
-#include "BatteryData.cpp"
+#include "BatteryData.h"
 #include <QTime>
-#include <QLocale>
-#include <cmath>
-#include <QTextStream>
 
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
-
+    const double HOUR_TO_MILLISECONDS = 3600000;
+    const double MINUTE_TO_MILLISECONDS = 60000;
+    const double SECOND_TO_MILLISECONDS = 1000;
 }
 
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
 {
+    currentAmphoursUsed_ = 0;
+    totalAmphoursUsed_ = (initialStateOfChargePercent_ * BATTERY_AMP_HOUR_CAPACITY)/100;
```

Comment:
Use initialization list to initialize your variables (see the initialStateOfChargePercent_)